cBioPortal/cbioportal-frontend
# mutational signatures implementation

**Changes included in this pr:**
1. Mutational signatures in the oncoprint tab
![image](https://user-images.githubusercontent.com/15748980/74272549-d3506980-4cdc-11ea-9ab5-75032e2bf87b.png)
2. Mutational signatures in the plots tab
![image](https://user-images.githubusercontent.com/15748980/74272571-de0afe80-4cdc-11ea-99df-face03b3a602.png)
3. Mutational signatures table in the patient view
![image](https://user-images.githubusercontent.com/15748980/74272636-fa0ea000-4cdc-11ea-9c04-72c129837632.png)

**How to test:**
1. go to https://triage.cbioportal.mskcc.org and select `msk_impact_2017` study.
2. press `F12` to open the browser console and paste this into console then enter: `localStorage.setItem("netlify", "deploy-preview-2653--cbioportalfrontend")`
3. check the new features in oncoprint tab (result view),  plots tab(result view) and Mutational Signature Data tab (patient view).
